docs/source/libraries-and-api/deprecations-list.md

```diff
@@ -13,8 +13,8 @@ framework, see {ref}`deprecation:list of the deprecated functionality in leapp`.
 Only the versions in which a deprecation has been made are listed.
 
 ## Next release <span style="font-size:0.5em; font-weight:normal">(till TODO date)</span>
-
-- Note: nothing new deprecated yet
+- Shared libraries
+  - **`leapp.libraries.common.config.get_distro_id()`** - The function has been replaced by variants for source and target distros - get_source_distro_id and get_target_distro_id.
```

Comment:
Since the `get_distro_id` has a special formatting, so should `get_source_distro_id` and `get_target_distro_id`. 
```suggestion
  - **`leapp.libraries.common.config.get_distro_id()`** - The function has been replaced by variants for source and target distros - `get_source_distro_id` and `get_target_distro_id`.
```